pkg/controller/servicebindingrequest/binder.go

```diff
@@ -469,24 +468,24 @@ func (b *Binder) update(objs *unstructured.UnstructuredList) ([]*unstructured.Un
 	updatedObjs := []*unstructured.Unstructured{}
 
 	for _, obj := range objs.Items {
-		// store a copy of the original object to later be used in a comparison
-		originalObj := obj.DeepCopy()
+		// modify the copy of the original object and use the original one later for comparison
+		updatedObj := obj.DeepCopy()
```

Comment:
It is not just about renaming. The original code takes the address obj and return it as result. And the address will be overwritten if there are mutiple objects in objs.Items. So I changes the behavior to modify a copy of obj to avoid the shadowing.

Reply:
Oh, thanks for clarifying!

In this case, it could also be possible to shadow `obj` [with itself](https://play.golang.org/p/TNVUm6RPalH) to avoid renaming in this unit:

```go
obj := obj
```

I'm fine with keeping the change though :)